### PR TITLE
Fix misra Rule 8.4

### PR DESCRIPTION
**Pull request by alfred2g**
Fix misra Rule 8.4

Description
-----------
Change the build system to add static a satisfy mira rule 8.4

Test Steps
-----------
Rule violation disappeared from the code checker

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/Coverity/ConfigFiles/FreeRTOSIPConfig.h [hunk 1]
```diff
@@ -30,7 +30,7 @@
 #ifndef FREERTOS_IP_CONFIG_H
 #define FREERTOS_IP_CONFIG_H
 
-#define _static
+#define _static static
```
Comment (AniruddhaKanhere) [inline test/Coverity/ConfigFiles/FreeRTOSIPConfig.h:33]: I think the correct way is to remove `_static` from files - and replace it with `static`.

We had added that to make CBMC work - but it is not needed anymore.

Can we please do that?

**Review by AniruddhaKanhere — COMMENTED** [review]
Thanks @alfred2g for the PR, I have a suggestion which should fix this problem once and for all. Can you please take a look?

File: test/Coverity/ConfigFiles/FreeRTOSIPConfig.h (hunk 1 above)
Comment (alfred2g) [reply test/Coverity/ConfigFiles/FreeRTOSIPConfig.h:33]: we use it for unit tests as well in some parts no ?

**Review by htibosch — APPROVED** [review]
Looks all good, thank you.

**Review by aggarg — APPROVED** [review]

**Pull request merged**